src/entities/Proposal/routes.ts

```diff
@@ -620,7 +621,8 @@ async function getGrantsByUser(req: Request): ReturnType<typeof getGrants> {
   let coauthoringProposalIds = new Set<string>()
 
   if (isCoauthoring) {
-    const coauthoring = await Governance.get().getProposalsByCoAuthor(address, CoauthorStatus.APPROVED)
+    const requests = await CoauthorModel.findProposals(address, CoauthorStatus.APPROVED)
+    const coauthoring = await filterCoauthorRequests(requests)
```

Comment:
I think it's not necessary to filter the requests due to they are only the approved ones 

<img width="844" alt="image" src="https://user-images.githubusercontent.com/45410089/192285473-c280451a-1720-4404-9a32-603e5d5f16cc.png">